site/webapp/pom.xml

```diff
@@ -63,7 +63,7 @@
                 </includes>
             </resource>
             <resource>
-                <filtering>true</filtering>
+                <filtering>false</filtering>
                 <directory>src/main/resources</directory>
```

Comment:
I take it you checked for any regressions.

Reply:
I did, everything seems fine.